File: duktape/combo/res/cjs/fsutils/fs/fs.js
console.log("fsutils/fs/fs.js:start:"+new Date().getTime());
var fs;
try{
	Duktape.fin(fsutils_fs.prototype,function(a){
		try {
			delete(a);
			console.log('finalizer, foo ->');//, x.foo);
		} catch (e) {
			console.log('WARNING: finalizer failed (ignoring): ' + e.toString());
		}
	});
	fs=new fsutils_fs();
	fs.test();
	fs=null;
	Duktape.gc();
}catch(e){
	console.error(e);
}
console.log("fsutils/fs/fs.js:end:"+new Date().getTime());

File: duktape/combo/src/app/duktape/util/wrappers/fsutils/fs.cpp
#include"app/duktape/wrappers/fsutils/fs.h"
#include<iostream>
namespace app::duktape::wrappers::fsutils{
	Fs::Fs(){
		std::cout<<"app::duktape::utils::wrappers::fsutils::Fs()"<<std::endl;
	}
	Fs::~Fs(){
		std::cout<<"app::duktape::utils::wrappers::fsutils::~Fs()"<<std::endl;
	}
	void Fs::test(){
		std::cout<<"app::duktape::utils::wrappers::fsutils::test()"<<std::endl;
	}
}
